### PR TITLE
add more packages to rosdep definitions

### PR DESCRIPTION
**Pull request by ftsell**
## Proposed changes
This PR adds a seconds wave of bitbots packages to our rosdep definitions as well as some dependency keys which are not yet present in upstream.

## Necessary checks
- [ ] Update package version
- [ ] Run `catkin build`
- [ ] Write documentation
- [ ] Create issues for future work
- [ ] Test on your machine
- [ ] Test on the robot
- [x] Put the PR on our Project board

### CODE REVIEW

**Review by timonegk — APPROVED** [review]

File: rosdep_misc.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+# rosdep definitions for required packages that are not yet present in the official rosdep definitions
+# upstream package definitions are hosted at https://github.com/ros/rosdistro/tree/master/rosdep
+
+beignet-dev:
+  ubuntu: beignet-dev
+  debian: beignet-dev
+libcv-bridge-dev:
+  ubuntu: libcv-bridge-dev
+  debian: libcv-bridge-dev
```
Comment (timonegk) [inline rosdep_misc.yml:10]: Wait, why is this necessary? There is ros-melodic-cv-brigde which is the correct package in my opininion

**Review by timonegk — CHANGES_REQUESTED** [review]

File: rosdep_misc.yml (hunk 1 above)
Comment (ftsell) [reply rosdep_misc.yml:10]: Because I did not find a rosdep key for cv bridge. I look [here](https://github.com/ros/rosdistro/tree/master/rosdep) and [here](https://github.com/ros/rosdistro/blob/master/melodic/distribution.yaml). If you feel I have overlooked it, please tell me the correct rosdep key for cv bridge.

Comment (timonegk) [reply rosdep_misc.yml:10]: I think you have to depend on `vision_opencv`: https://github.com/ros/rosdistro/blob/master/melodic/distribution.yaml#L13953

Comment (ftsell) [reply rosdep_misc.yml:10]: Ahh I got it now. I misinterpreted how the [distribution.yaml](https://github.com/ros/rosdistro/blob/master/melodic/distribution.yaml) file works and now see that there actually is an existing `cv_bridge` key
